Library/Psionics/Psionics Advantages.adq

```diff
@@ -3,6 +3,69 @@
 	"version": 2,
 	"id": "4a1c9802-f8dc-43bc-a1df-3750b319427b",
 	"rows": [
+		{
+			"type": "advantage",
+			"id": "84aafd42-e3cc-4a2d-a717-7cf189907afc",
+			"name": "Enhanced Power Defence",
```

Comment:
Does the book actually spell this "Defence" and not "Defense"?

Reply:
I didn't actually add this. I merely alphabetized the list. I checked the book, and it is indeed simply a typo, so I fixed it.

Reply:
Thanks